### PR TITLE
Fix tests and move to github actions

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
@theofidry I'd like to try to fix the build, but TravisCi is not running.
What am I doing wrong ?

**Comment by theofidry** [comment]
I think it's just Travis hitting it's cap... It seriously reduced its OSS support recently

**Comment by VincentLanglet** [comment]
> I think it's just Travis hitting it's cap... It seriously reduced its OSS support recently

The best would be to migrate to github actions.

**Comment by VincentLanglet** [comment]
Hi @theofidry,

I moved the CI to github actions and fixed the tests: https://github.com/VincentLanglet/AliceDataFixtures/runs/1622755518

File: vendor-bin/eloquent/composer.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
         ]
     },
     "require-dev": {
-        "illuminate/database": "~5.5.0",
-        "illuminate/filesystem": "~5.5.0",
+        "illuminate/database": "~5.5.0 || ~6.20.0 || ~7.29.0",
+        "illuminate/filesystem": "~5.5.0 || ~6.20.0 || ~7.29.0",
```
Comment (theofidry) [inline vendor-bin/eloquent/composer.json:9]: we can drop v5 & v6 here to make it easier

**Review by theofidry — APPROVED** [review]
👌 a tiny change but looks really good; thanks a lot for that work

File: vendor-bin/eloquent/composer.json (hunk 1 above)
Comment (VincentLanglet) [reply vendor-bin/eloquent/composer.json:9]: Can we ? The v7 is only working with Symfony 5 to me https://github.com/illuminate/filesystem/blob/7.x/composer.json

Comment (VincentLanglet) [reply vendor-bin/eloquent/composer.json]: Done !

**Comment by theofidry** [comment]
Thanks!

**Comment by VincentLanglet** [comment]
You're welcome @theofidry.

Could a 1.3.2 release be possible ?

**Comment by theofidry** [comment]
I'll do as soon as the build is finished and I can protect the branch

**Comment by theofidry** [comment]
hm I think there is something wrong with the build as it's been over 40m already https://github.com/theofidry/AliceDataFixtures/runs/1629975921?check_suite_focus=true

**Comment by VincentLanglet** [comment]
Indeed the last commit didn't work: https://github.com/VincentLanglet/AliceDataFixtures/runs/1628030760?check_suite_focus=true

I tried to revert the change you asked https://github.com/theofidry/AliceDataFixtures/pull/161

**Pull request merged**